migrations/anser.go

```diff
@@ -70,6 +69,7 @@ type migrationGeneratorFactory func(anser.Environment, migrationGeneratorFactory
 // execution. See the anser documentation and the
 // anser/example_test.go for an example.
 func (opts Options) Application(env anser.Environment, evgEnv evergreen.Environment) (*anser.Application, error) {
+	const AllLogCollection = "event_log"
```

Comment:
I assume this is a lint error, and should be allLogCollection

Reply:
done